presto-spi/src/main/java/com/facebook/presto/spi/type/SqlTimestampWithTimeZone.java

```diff
@@ -81,8 +82,7 @@ public boolean equals(Object obj)
     @Override
     public String toString()
     {
-        SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS");
-        format.setTimeZone(getTimeZoneForKey(timeZoneKey));
-        return format.format(new Date(millisUtc)) + " " + timeZoneKey.getId();
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSS VV");
+        return ZonedDateTime.ofInstant(Instant.ofEpochMilli(millisUtc), ZoneId.of(timeZoneKey.getId())).format(formatter);
```

Comment:
`Instant.ofEpochMilli(...).atZone(...)` instead of `ZonedDateTime.ofInstant(Instant.ofEpochMilli(...), ...)`